Javaagent release 8.7.0 [DO NOT MERGE UNTIL 10/26] (#14984)
newrelic/docs-website: src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx

In addition to overriding configuration settings, the following system properties are recognized by the agent:

<CollapserGroup>
  <Collapser
    id="com-newrelic-jboss-jsr77-fix"
    title="com.newrelic.jboss.jsr77.fix"
  >
    <table>
      <tbody>
        <tr>
          <th>
            Type
          </th>

          <td>
            Boolean
          </td>
        </tr>

        <tr>
          <th>
            Default
          </th>

          <td>
            (none)
          </td>
        </tr>
      </tbody>
    </table>

>     Starting from agent version `8.7.0`, if set to true and the Agent is running with JBoss EAP / Wildfly, 
>     this will prevent the agent from requiring any class under the package `javax.management` to be loaded by the system classloader.
>     This is added to help fix application startup issues with JBoss Modules when the J2EE/Jakarta Management API is used. 
>     More information is provided in the JBoss / Wildfly [troubleshooting doc](/docs/apm/agents/java-agent/troubleshooting/classloading-issues-from-jboss-and-wildfly).

Reviewer: ```suggestion
    Starting from agent version `8.7.0`, if set to true and the agent is running with JBoss EAP / Wildfly, 
    this will prevent the agent from requiring any class under the package `javax.management` to be loaded by the system classloader.
    This is added to help fix application startup issues with JBoss Modules when the J2EE/Jakarta Management API is used. 
    For more information see the [JBoss / Wildfly troubleshooting doc](/docs/apm/agents/java-agent/troubleshooting/classloading-issues-from-jboss-and-wildfly).
```

Reviewer: Hi @obenkenobi , this docs link is not valid. Is there another existing page you had in mind?

Author: Merged some code from another [PR](https://github.com/newrelic/docs-website/pull/14900) I made. This was where the link came from. Both were intended for the 8.7.0 release so it makes sense to merge the 2 now.
